### ISSUE TITLE
Investigate the SBT test failures in the strongbox-web-integration-tests

### ISSUE THREAD

**Issue by carlspring** [strongbox/strongbox#1479]
# Task Description

We are getting the following error in the SBT tests of the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/):

```
[INFO] Using Groovy 2.4.7 to perform execute.
Test test-sbt-common-flow.groovy

Execute command[s]: 

sbt compile

Copying runtime jar.

gave the following error: 

[ERROR] /java/sbt-1.1.6/bin/sbt-launch-lib.bash: line 258: /java/jdk8u202-b08/bin/java: No such file or directory

mkdir: cannot create directory ??????: No such file or directory
/java/sbt-1.1.6/bin/sbt-launch-lib.bash: line 73: /java/jdk8u202-b08/bin/java: No such file or directory
/java/sbt-1.1.6/bin/sbt-launch-lib.bash: line 73: /java/jdk8u202-b08/bin/java: No such file or directory
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 48.860 s (Wall Clock)
[INFO] Finished at: 2019-09-27T06:41:20Z
[INFO] ------------------------------------------------------------------------
[INFO] [jenkins-event-spy] Generated /home/jenkins/agent/workspace/trongbox_builds_strongbox_master/strongbox-web-integration-tests/sbt@tmp/withMaven73fa744e/maven-spy-20190927-064031-121955445287199365862.log
[ERROR] Failed to execute goal org.codehaus.gmavenplus:gmavenplus-plugin:1.6:execute (sbt-integration-tests) on project strongbox-web-integration-tests-sbt: Error occurred while calling a method on a Groovy class from classpath. InvocationTargetException: assert !proc.exitValue()
[ERROR]        ||    |
[ERROR]        ||    127
[ERROR]        |java.lang.UNIXProcess@2d93df9
[ERROR]        false
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

The issue will need to be investigated and the fix for this will have to be done in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/).

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

**Comment by steve-todorov** [comment]
@mslowiak Just FYI - this issue is related to some updates I did on our infrastructure. I'll fix it later today and restart your PR #1478. :)

**Comment by steve-todorov** [comment]
This is now solved. Thanks for reporting @carlspring !
